<!DOCTYPE html>
<html>
<head>
<title>Time Series Analysis of Eurasian Selective Pressure During the Neolithic Revolution</title>
</head>
<body>
<article>
<center>
	<header>
		<h1>Time Series Analysis of Eurasian Selective Pressure During the Neolithic Revolution</h1>
		<h2>Nathan Oakes</h2>
		<h3>5/20/2016</h3>
	</header>
</center>
<div>
<h3>Introduction</h3>
<p>
	The study of human population genetics has historically been an endeavor to infer evolutionary and demographic changes through time from data standing genetic variation between and within modern populations. If a temporal dimension can be added to the population data there could be significant improvement to current demographic modeling, allele age estimation and tests of natural selection [1]. Unfortunately this has been out of reach for many scientists interested in human evolutionary history due to prohibitive costs to sequencing, technical challenges of sequencing and analyzing ancient DNA, and the small pool of samples with enough preserved DNA to be sequenced.  However, recently 230 ancient Eurasians have been sequenced [2], and that has data made public, so that potential for time series analysis of human populations can be made possible. Two exciting outcomes came from this publication. The first being that the ancient samples were dated from 7 to 1 thousand years ago, spanning the emergence of agriculture through Europe, which had such a strong impact on the environment in which these humans lived that there is evidence of some of the strongest signatures of selection originating from this period [3]. The second being that the authors found that modern Eurasians populations can be modeled approximately well as being a linear combination of a collection of ancient populations. From this dataset I hope to use a method of analysis that leverage the time component of frequency changes over time to infer signatures of selection.
	</p>
<h3>Method</h3>
	<p>
	Given that the modern CEU population can be modeled as a linear combination of 19.6% HG, 25.7% EF, 54.7% SA populations, for each allele I calculated the frequencies of these combined ancient Eurasian populations, and similarly calculated a time point that is, in the same proportions, a linear combination of the dates the samples are said to be from, which was 3715.7 years ago or ~155 generations. So with the modern and ancient allele frequencies  I construct a hidden markov model to calculated the selection coefficient most likely to produce the shift in allele frequencies conditional on a super-exponential population grown model [4]. 
	</p>
	<p>
	Hidden markov models leverage what are called markov chains, which are random processes that transition from one state to the next bases only on the current state. In this case, the probability that an allele in the population at frequency x at one generation will be at frequency y in the next generation is modeled as binomial sampling with the probability of success equal to x times the coefficient of selection, and the number of trials equal to N(t) at t1. In this way an allele's progression over time may be modeled as a markcov chain, and is the foundation of the delta approximation[5] which is used to estimate these transition probabilities. This isn't the whole story, however, because the samples available are not the true frequencies that exist , the true frequencies are “hidden”, so this uncertainty is represented as an additional binomial sampling step where the probability of success is equal to the a guess at the true allele frequency. 
	</p>
	<script src="https://raw.githubusercontent.com/NathanCompBioakes/NathanCompBioakes.github.io/master/mml-math-3.gif"></script>
	<p>
	The following code illustrates the method implemented in C++.
	</p>
	<script src="https://gist.github.com/NathanCompBioakes/7cce0e23a060773c56410ac22b8f1803.js"></script>

<h3>Results</h3>
<center>
<div>
	<img src="https://raw.githubusercontent.com/NathanCompBioakes/NathanCompBioakes.github.io/master/plot.png", align="middle">
</div>
</center>
<h3>References</h3>
<p>
    [1] Sams, et al. The utility of ancient human DNA for improving allele age estimates, with implications for demographic models and tests of natural
    selection. Journal of Human Evolution (Feb 2015)
</p>
<p>
    [2] Mathieson, et al.Genome-wide patterns of selection in 230 ancient Eurasians. Science (24 Dec 2015)
</p>
<p>
    [3] Gerbault, et al. Evolution of lactase persistence: an example of human niche construction. Philos Trans R Soc Lond B Biol Sci (27 Mar 2011)
</p>
<p>
    [4] Inference of Super-exponential Human Population Growth via Efficient Computation of the Site Frequency Spectrum for Generalized Models. Genetics. (Jan 201
</p>
<p>
    [5] Lacerda, et al. Population Genetics Inference for Longitudinally-Sampled Mutants Under Strong Selection. Genetics (Sept 10, 2014)
</p>
</div>
</article>
</body>
</html>

